src/main/java/de/embl/cba/bdv/ome/zarr/loaders/N5OMEZarrImageLoader.java

```diff
@@ -132,7 +147,29 @@ private void fetchSequenceDescriptionAndViewRegistrations()
 			for ( int setupId = 0; setupId < numSetups; setupId++ )
 			{
 				ViewSetup viewSetup = createViewSetup( setupId );
-				int setupTimepoints = ( int ) setupToAttributes.get( setupId ).getDimensions()[ T ];
+				int setupTimepoints = 1;
+				is5D = false;
+				if (setupToAttributes.get(setupId).getNumDimensions() > 4 && axesMap.containsKey("t")) {
```

Comment:
if we have 5 axes we know that axes are "tczyx", so you only need the first "if", not the second "else if". (The conditions are identical.)